advancedbrowser/advancedbrowser/core.py

```diff
@@ -107,13 +108,30 @@ def columnData(self, index):
         if type in self.browser.customTypes:
             return self.browser.customTypes[type].onData(c, n, type)
 
-    def search(self, txt):
+    def search(self, txt, reset=True):
```

Comment:
What does this reset do? Is it left over from the older version?

Reply:
It is. Thank you for noticing. Line 118 of 2.0/aqt/browser.py code was
 def search(self, txt, reset=True): 
(in class datamodel). So I had to keep this argument in my add-on code.
In 2.1 it seems this option disappeared. It's not needed in the add-on either. I'm going to push a version without this argument.

Reply:
I had a look and it seems the 2.1 version always does a begin/end reset at the start/end of the function, so it will be a good idea to keep that but remove the flag that makes it optional.

Reply:
Done

Reply:
Looks like you've moved the file up a directory.

Reply:
Corrected